pinot-core/src/test/java/org/apache/pinot/queries/BooleanNullEnabledQueriesTest.java

```diff
@@ -194,21 +194,6 @@ public void testQueries() {
         }
       }
     }
-    {
```

Comment:
Why removing this test?

Reply:
We removed proper null handling logic in equality/inequality/membership operators from this PR, and as a result removed this test (related to a null check in value matcher)

Reply:
Same here

Reply:
This is the plan, to remove now and add it as part of the PR handling nulls in. equality/inequality/membership operators.